### ISSUE TITLE
Instant Snake Bookmarklet?

### ISSUE THREAD

**Issue by Erisfiregamer1** [DarkSnakeGang/GoogleSnakeModLoader#17]

**Comment by Yarmiplay** [comment]
We thought about making a kind of copy of the game and running on that, first issue that comes to mind is that it would be stuck in the version we copy from so that's not future proof if any updates ever come out - but it would avoid needing to install tampermonkey. It would also mean hosting it somewhere else like Github Pages (not a big deal). That being said, we don't have a proper way of making this copy for the simple fact we don't know how, I've personally tried some methods and didn't get anything that works. Archive.org does have a working method so it's possible, but we need to somehow do it ourselves so we can avoid the bookmark as well (it'll just load everything right there on the website without a bookmark - that'd be ideal).

**Comment by Erisfiregamer1** [comment]
Nobody would mind if it's outdated. Having a bookmarklet method of running it would be perfect for everyone.

All it would have to do is kill the old Google Snake instance and load the (modified) version. More stable too since we can manually patch it.

**Comment by PapyrusWasTaken** [comment]
please do this. I cant use tampermonkey because of my school chromebook

**Comment by Erisfiregamer1** [comment]
That, too.

**Comment by Yarmiplay** [comment]
If you guys give us a proper way to copy the entire web page in a way to host it independently, that'd pass the technical barrier we are facing at the moment regarding this.
We'll be happy to do this if we can get passed this barrier.

**Comment by Erisfiregamer1** [comment]
Figured it out?

**Comment by hwatson381** [comment]
> Figured it out?

Yes! https://googlesnakemods.com/ We have a website now that's a clone of google snake with mods, but also with some tweaks to make it work on mobile as well.

**Comment by Erisfiregamer1** [comment]
Hell yeah! Now I can play Google Snake's More Menu without needing an extension again! :D
